tests/integration/java-integration/run_local_build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -ueo pipefail
+
+SCRIPT_ROOT="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
+
+CONTAINER_RUNTIME=${CONTAINER_RUNTIME:-docker}
+"$CONTAINER_RUNTIME" run \
+ -t \
+ --rm \
+ -e "CONTAINER_RUNTIME=${CONTAINER_RUNTIME}" \
+ -e FLUENTBIT_COMMAND="/fluent-bit -c /usr/src/mymaven/src/test/resources/fluentbit.conf" \
+ -e MAVEN_CONFIG=/var/maven/.m2 \
+ -u "$(id -u):$(id -g)" \
+ -v "$SCRIPT_ROOT/../../../build/bin/fluent-bit":/fluent-bit \
```

Comment:
Probably should check we have built this first, no point even attempting anything if we do not have the binary.

I would also aim to use read-only mounts as much as possible, add `:ro` to the end of the volume mount command.